scripts/blocklist-skylink.sh

```diff
@@ -29,25 +29,17 @@ else
     skylinks=("$1") # just single skylink passed as input argument
 fi
 
-####################################################
-# iterate through all servers and block the skylinks
-####################################################
+#########################################################################
+# iterate through all servers, block the skylinks and purge it from cache
+#########################################################################
 for server in "germany.siasky.net" "helsinki.siasky.net" "us-west.siasky.net" "us-va-1.siasky.net" "us-pa-1.siasky.net" "us-pa-2.siasky.net" "siasky.xyz";
 do
-    #############################################################
-    # iterate throught all skylinks and add each one to blocklist
-    #############################################################
     for skylink in "${skylinks[@]}";
     do
         echo ".. ⌁ Blocking skylink ${skylink} on ${server}"
-        ssh -q -t user@${server} 'docker exec sia siac skynet blocklist add '$skylink''
-    done
 
-    ######################################################
-    # purge nginx cache after all the skylinks are blocked
-    ######################################################
-    ssh -q -t user@${server} 'docker exec nginx sh -c "rm -rf /data/nginx/cache/*"'
-    echo ".... 🗑️ Pruned nginx cache on ${server}"
+        ssh -q -t user@${server} "docker exec sia siac skynet blocklist add $skylink && docker exec nginx curl -s -i -X PURGE http://localhost/$skylink | egrep \"^(OK|HTTP|X-)\""
```

Comment:
Have you tested this? At first glance it seems like the whole `&& docker exec nginx...` command will be passed as parameters to the `siac` call in the first docker command.

Do we have any reporting of errors? I see you're filtering out the OK status but I don't know what happens with the not-OK statuses.

Reply:
I did test it out.

Example of successful (one server, skylink valid, redacted):
```bash
> scripts/blocklist-skylink.sh XXX

.. ⌁ Blocking skylink XXX on siasky.xyz
Skynet Blocklist updated
HTTP/1.1 200 OK
X-Purged-Count: 0
OK
✓ All done !
```

Example of failure:  (one server, skylink invalid, redacted):
```bash
> scripts/blocklist-skylink.sh XXXINVALID

.. ⌁ Blocking skylink XXXINVALID on germany.siasky.net
Unable to update skynet blocklist: [POST request error; error parsing skylink: [unable to decode skylink; skylink has incorrect size]]
```

Reply:
Great, I just wanted to make sure because that linking of docker commands looked very suspicious.